WordPress/Classes/ViewRelated/Blog/SharingViewController.m

```diff
@@ -123,6 +123,17 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
     }
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    PublicizeService *publicizer = self.publicizeServices[indexPath.row];
```

Comment:
Getting a crash here when there the publicizeServices array is empty.  This can be reproduce by logging out, logging back in and viewing the Sharing vc for the first time. 
<img width="1499" alt="screen shot 2017-07-14 at 9 47 53 am" src="https://user-images.githubusercontent.com/1435271/28217241-1883e6c0-687a-11e7-81f4-3591a7c3fb72.png">

This seems to fix it:
```
if ([self.publicizeServices count] > 0) {
        PublicizeService *publicizer = self.publicizeServices[indexPath.row];
        NSArray *connections = [self connectionsForService:publicizer];
        if ([publicizer.serviceID isEqualToString:PublicizeService.googlePlusServiceID] && [connections count] == 0) { // Temporarily hiding Google+
            return 0;
        }
    }
    return UITableViewAutomaticDimension;
```